client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
 				<button
 					className="conversation-caterpillar__count"
 					onClick={ this.handleTickle }
-					title={
-						commentCount > 1
-							? translate( 'View comments from %(commenterName)s and %(count)d more', {
-									args: {
-										commenterName: lastAuthorName,
-										count: commentCount - 1,
-									},
-								} )
-							: translate( 'View comment from %(commenterName)s', {
-									args: {
-										commenterName: lastAuthorName,
-									},
-								} )
-					}
+					title={ translate(
+						'View %(count)s more comments for this post',
```

Comment:
i think these are backwards (see the bottom of https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/caterpillar-copy)

Reply:
Good spot! Fixed.